uxarray/plot/utils.py

```diff
@@ -33,3 +35,30 @@ def reset_mpl_backend(self):
 
 # global reference to holoviews backend utility class
 backend = HoloviewsBackend()
+
+
+def great_circle(p1, p2, num_points=100):
+    """
+    Generate points along the great circle between two points.
+    p1 and p2 are arrays/lists of Cartesian coordinates.
+    """
+    # Normalize the input points
+    p1 = p1 / np.linalg.norm(p1)
+    p2 = p2 / np.linalg.norm(p2)
+
+    # Compute the angle between the points
+    omega = np.arccos(np.clip(np.dot(p1, p2), -1.0, 1.0))
```

Comment:
we already have the function that calculates the angle called `angle_between_vectors` right? That one uses a specific math formula to avoid some potential cancellation errors.

Reply:
Good catch, didn't think to use that one here.